.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Implements [DHIS2-XXXX](https://dhis2.atlassian.net/browse/DHIS2-XXXX)
+
+---
+
+### Key features
+<!-- Remove if not applicable -->
+1. _feature_
+
+---
+
+### Description
+
+_text_
+
+---
+
+### BREAKING CHANGE
+<!-- Remove if not applicable -->
+_reason_
+
+---
+
+### TODO
+
+-   [ ] Write Documentation
+-   [ ] Write Tests
+-   [ ] _task_
+
```

Comment:
Maybe we can turn this into a "Checklist" that has to be completed (although it's not verified technically):

```suggestion
### Checklist

-   [ ] Write Documentation
    - <!-- If not needed, explain why, otherwise remove this bullet point -->
-   [ ] Write Tests
    - <!-- If not needed, explain why, otherwise remove this bullet point -->
-   [ ] Actual task completed
```

---

Does it make sense to change "_\_task_\_" to "Actual task completed" so it doesn't have to be changed? Not sure, just an idea.